tests/suites/test_suite_ecp.data

```diff
@@ -1026,3 +1026,195 @@ ecp_mod_setup:"fffffffffffffffffffffffe26f2fc17f69466a74defd8d":MBEDTLS_ECP_DP_C
 ecp_setup_negative_test #28 Invalid Curve Type
 depends_on:MBEDTLS_ECP_DP_CURVE448_ENABLED
 ecp_mod_setup:"fffffffffffffffffffffffe26f2fc17f69466a74defd8d":MBEDTLS_ECP_DP_NONE:MBEDTLS_ECP_MOD_SCALAR:MBEDTLS_ERR_ECP_BAD_INPUT_DATA
+
+ecp_mul_inv #1 MBEDTLS_ECP_MOD_SCALAR(MBEDTLS_ECP_DP_SECP192R1)
+depends_on:MBEDTLS_ECP_DP_SECP192R1_ENABLED
+ecp_mod_mul_inv:"0000000000000000000000000000152d02c7e14af67fe0bf":MBEDTLS_ECP_DP_SECP192R1:MBEDTLS_ECP_MOD_SCALAR:0
```

Comment:
How is that data generated? Will there be follow-up commits which will enable it on the python test framework?

Reply:
I think that these tests are not very convenient to do in the test framework. We would need to extend the framework to accommodate them. I think it is easier from both writing, review and maintenance perspective if we just go for manual test cases.

Reply:
If we want to capture how these were generated, I suggest we add a comment in the .data file to the test case.